mono/mini/debugger-agent.c

```diff
@@ -930,9 +930,13 @@ mono_debugger_get_generate_debug_info ()
 }
 
 MONO_API void
-mono_debugger_disconnect (const char *message)
+mono_debugger_disconnect ()
```

Comment:
Why did the function signature change?

Reply:
I'm just making it match what it already is on the Unity side.